my-dsl.md

```diff
@@ -2,37 +2,60 @@
 _What is the name of the language? Link the name to its webpage 
 (if appropriate)._
 
+The language is called Scratch. Its webpage is [here](https://scratch.mit.edu/ "Scratch's Homepage")
+
 
 # Domain
 _Describe the language's domain in five words._
 
+Game creation for CS education. 
+
 
 # Computational model
 _What is the underlying computational model of this language? To answer this 
 question, provide a high-level description (no more than 100 words) of the 
 computation that occurs when someone executes a program in this language._
 
+Scratch is an event based programming language, so all computational logic is triggered by specific events, such as when the green flag (effectively the play button) is pressed, or when two sprites touch. When a particular event occurs, the code block connected to it runs. This could involve, for example, moving a sprite 10 units to the right, changing the sprite's costume (visual appearence), and then broadcasting "done moving". This broadcast (which I think of as similar to callbacks in JavaScript) can then trigger other events in the application. 
```

Comment:
I haven't tried it myself, but I can't think of a reason why that wouldn't be okay. I believe Scratch allows parallelization, so having a single event kick off multiple processes should be fine.